### PR TITLE
Update docs

### LINKED ISSUE #68: Update User Guide
- [x] Fatin: 1, 2
- [x] YiJing: 3
- [x] Sam: 4.1 - 4.2
- [x] Gabe: 4.3
- [x] Rai: 4.4 - 4.7

### LINKED ISSUE #69: Update Developer Guide
- [x] A: YiJing
- [x] B: Rai
- [x] C: Gabe, Fatin
- [x] D, E: Sam

### PR DESCRIPTION
**Pull request by fatin99**

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -4,53 +4,53 @@
...
-=== Johnny Roe
-image::m133225.jpg[width="150", align="left"]
-{empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
+=== FATIN NABILAH BTE SUHAIMI
+image::fatin99.png[width="150", align="left]
+{empty}[https://github.com/fatin99[github]] [<<johndoe#, portfolio>>]
 
 Role: Developer +
-Responsibilities: Dev Ops + Threading
+Responsibilities: Documentation  and Code Quality
```
Comment (PotatoCombat) [inline docs/AboutUs.adoc]: Extra space!

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -316,14 +316,158 @@ _{More to be added}_
...
+=== Use case: Viewing a page
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to view a different page
+2. AddressBook layout changes to show the new page
```
Comment (PotatoCombat) [inline docs/DeveloperGuide.adoc]: TA-Tracker instead of AddressBook!!

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -14,34 +14,47 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/se-edu/addressbook-level3
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team W17-4`      Since: `Jun 2016`      Licence: `MIT`
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Since: `Jan 2020`

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -14,34 +14,47 @@ ifdef::env-github[]
...
 
 == Introduction
 
-AddressBook Level 3 (AB3) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB3 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+TATracker (TAT) is a productivity tool made for NUS Computing Teaching Assistants (TAs) who want to be able to track and manage all of their claimable hours in NUS. Interested? Jump to Section 2 “Quick Start” to get started!
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Maybe TA-Tracker for the product name?

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -316,14 +316,158 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
```
Comment (PotatoCombat) [inline docs/DeveloperGuide.adoc]: System is TA-Tracker

File: docs/UserGuide.adoc (hunk 4 above)
Comment (PotatoCombat) [reply docs/UserGuide.adoc]: Use the hyper link! Jump to <<Quick Start>> to get started!

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -316,14 +316,158 @@ _{More to be added}_
...
+
+[discrete]
+=== Use case: Edit Student
+
+*MSS*
+
+1. User requests to list students
+2. AddressBook shows a list of students
+3. User requests to edit a specific student in the list
+4. AddressBook edits the student according to the specified parameters
```
Comment (aakanksha-rai) [inline docs/DeveloperGuide.adoc]: It says AddressBook everywhere, need to change it to say TA-Tracker

File: docs/DeveloperGuide.adoc [hunk 7]
```diff
@@ -316,14 +316,158 @@ _{More to be added}_
...
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: Change the hourly pay rate
+
+*MSS*
+
+1.  User requests to change the hourly pay rate to a specified amount
+2. AddressBook changes the pay rate
+3. AddressBook shows an edited TSS claims page the total pay adjusted to reflect the new pay rate
```
Comment (aakanksha-rai) [inline docs/DeveloperGuide.adoc]: You put full stops some places but don't in other places. Can make it more uniform?

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -50,6 +63,32 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 * Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 * Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 * Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+
+*Common Parameters*
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc:69]: Maybe the capped params can be quite?

eg. `PAGE_NAME`

File: docs/UserGuide.adoc [hunk 9]
```diff
@@ -14,34 +14,47 @@ ifdef::env-github[]
...
-* *`exit`* : exits the app
 
 .  Refer to <<Features>> for details of each command.
 
-[[Features]]
-== Features
+[[Layout]]
+== Layout
+
+. [.underline]#Dashboard:#
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Underline didn't work! Maybe use the normal header?
eg.
=== Dashboard

**Review by PotatoCombat — COMMENTED** [review]
Just need to change format and a few words!

**Review by aakanksha-rai — APPROVED** [review]
Looks good!

File: docs/UserGuide.adoc [hunk 10]
```diff
@@ -14,34 +14,49 @@ ifdef::env-github[]
...
 .  Type the command in the command box and press kbd:[Enter] to execute it. +
 e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
-
-* *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
-* **`delete`**`3` : deletes the 3rd contact shown in the current list
-* *`exit`* : exits the app
 
 .  Refer to <<Features>> for details of each command.
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Can change this to link to << Layout >> also? (No spaces)

**Review by PotatoCombat — COMMENTED** [review]
Just need to change the features link to layout, so that the hyperlink is correct.

**Review by PotatoCombat — APPROVED** [review]

**Pull request merged**